crates/red_knot_python_semantic/src/types/mro.rs

```diff
@@ -350,8 +350,9 @@ impl<'db> ClassBase<'db> {
     /// Attempt to resolve `ty` into a `ClassBase`.
     ///
     /// Return `None` if `ty` is not an acceptable type for a class base.
-    fn try_from_ty(ty: Type<'db>) -> Option<Self> {
+    fn try_from_ty(db: &'db dyn Db, ty: Type<'db>) -> Option<Self> {
         match ty {
+            Type::TypeAlias(alias) => Self::try_from_ty(db, alias.value(db)),
```

Comment:
The spec doesn't really clarify, but mypy and pyright both agree that a type alias isn't valid as a base class, and it won't work at runtime, either. So we don't need to support it in this way, it can go below in the long list of things that return `None`. (Though it will probably go inside the `KnownInstance` match instead.)